tags/feature.tag

```diff
@@ -2,5 +2,4 @@ type: text
 
 ---
 
-**Request a feature here:**
- <https://github.com/GeyserMC/Geyser/issues/new?assignees=&labels=&template=feature_request.md&title=>
\ No newline at end of file
+Request a feature here: https://github.com/GeyserMC/Geyser/issues/new?assignees=&labels=&template=feature_request.md&title=
```

Comment:
Why remove the no embed tags? This takes up a lot of space and isn't helpful:
![image](https://user-images.githubusercontent.com/13787163/99144940-32734e00-2662-11eb-9b67-d363d28aead6.png)

Reply:
Because we render this as an embed now.
![image](https://user-images.githubusercontent.com/5401186/99146272-6fddd880-266e-11eb-9a24-f6277d12af8d.png)